build/i2c/src/lib.rs

```diff
@@ -1261,7 +1310,8 @@ pub fn codegen(disposition: Disposition) -> Result<()> {
 pub struct I2cDeviceDescription {
     pub device: String,
     pub description: String,
-    pub num_measurement_channels: usize,
+    // (name, kind, id)
+    pub sensors: Vec<(Option<String>, Sensor, usize)>,
```

Comment:
...and here (I think this is the same tuple as above)

Reply:
Rolled in with the others in 5bf5ae8b